node/tessel-export.js

```diff
@@ -408,11 +410,16 @@ Tessel.Port.PATH = {
   'B': '/var/run/tessel/port_b'
 };
 
-Tessel.Pin = function(pin, port, interruptSupported, analogSupported) {
+Tessel.adcCompatiblePins = [4, 7];
+Tessel.pullCompatiblePins = [2, 3, 4, 5, 6, 7];
+Tessel.interruptCompatiblePins = [2, 5, 6, 7];
```

Comment:
Sorry to nitpick on this...

- Do these need to be exposed at all?
  - If yes, 
    - `Tessel.Pin.*` is a more appropriate home
    - `Compatible` isn't really the word that describes this information, these are more like a "capability", but honestly...
- I don't think these need to be exposed.